lectures/inventory_dynamics.md

```diff
@@ -27,6 +27,12 @@ kernelspec:
 ```{index} single: Markov process, inventory
 ```
 
+In addition to what's included in base Anaconda, we need to install the following packages
+
+```{code-cell} ipython3
+!pip install quantecon
```

Comment:
```suggestion
:tags: [hide-output]
!pip install quantecon
```

Reply:
thanks @jstac good call. I can take a look tomorrow at the numba warning 👍

Reply:
Thanks @mmcky  :-)